### DISCUSSION TITLE
monorepo:sentry-javascript should also include getsentry/sentry-electron

### DISCUSSION THREAD

**Discussion by markspolakovs** [renovatebot/renovate#30163] [Suggest an Idea]
### Tell us more.

Sentry's Electron SDK is [often combined](https://docs.sentry.io/platforms/javascript/guides/electron/#framework-specific-sdks) with JavaScript framework specific SDKs, however they're shipped in two separate repos, https://github.com/getsentry/sentry-electron and https://github.com/getsentry/sentry-javascript.

The versions of the two are tightly coupled and often lead to TypeScript errors if they don't match (e.g. https://github.com/ystv/badger/pull/534 needed me to manually upgrade `@sentry/react` to match `@sentry/electron`).

It might be worth adding getsentry/sentry-electron to the sentry-javascript monorepo, or making a separate monorepo with them both, so that upgrades to the two are grouped.

**Comment by rarkins** [comment]
The downside is that if releases aren't in sync then you end up with "immortal" PRs
